src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -184,6 +185,21 @@
 						TextTrimming="CharacterEllipsis"
 						TextWrapping="Wrap" />
 				</Grid>
+				<CheckBox
+					x:Name="SelectionCheckbox"
+                    x:Load="False"
+					Grid.RowSpan="2"
+					Grid.ColumnSpan="2"
+					Width="32"
+					MinWidth="0"
+					MinHeight="0"
+					Margin="6"
+					Padding="0"
+					HorizontalAlignment="Left"
+					VerticalAlignment="Top"
+					Checked="ItemSelected_Checked"
+					Unchecked="ItemSelected_Unchecked"
+					Visibility="{Binding ElementName=PageRoot, Path=AppModel.ShowSelectionCheckboxes, Converter={StaticResource BoolToVisibilityConverter}, Mode=OneWay}" />
```

Comment:
Will this also work for the one in the item template on line 238?

Reply:
Not sure what exactly you mean, both use the same visibility binding IIRC. Unfortunately though, I had to remove the x:Load setter on both since it seemed to just did not render them.